packages/terra-action-header/src/orion-fusion-theme/ActionHeaderContainer.module.scss

```diff
@@ -0,0 +1,37 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .orion-fusion-theme {
+    // .flex-header
+    --terra-action-header-align-items: center;
+    --terra-action-header-background-color: #f4f4f4;
+    --terra-action-header-border-bottom: 0.16666667rem solid #64696c;
+    --terra-action-header-border-left: none;
+    --terra-action-header-border-right: none;
+    --terra-action-header-border-top: none;
+    --terra-action-header-box-shadow: none;
+    --terra-action-header-min-height: 2.83333rem;
+    --terra-action-header-padding: 0;
+
+    // .flex-collapse
+    --terra-action-header-flex-collapse-align-self: center;
+    --terra-action-header-flex-collapse-margin: 0.5rem;
+
+    // .flex-fill
+    --terra-action-header-flex-fill-align-self: center;
+
+    // .flex-end
+    --terra-action-header-flex-end-align-self: center;
```

Comment:
Updated to `center` from [`stretch`](https://github.com/cerner/terra-core/pull/2921/files#diff-88dce3fa259e93fb4378f904ec7774c0L15) based on this [conversation](https://github.com/cerner/terra-clinical/pull/641#discussion_r405763957)